src/style.css

```diff
@@ -1 +1,65 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header__menu {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.header__icon {
+  margin-top: 10px;
```

Comment:
```suggestion
.header__menu {
  padding: 0 50px;
  display: flex;
  justify-content: space-between;
  align-items: center;
}

.header__icon {
  line-height: 0; /* to avoid extra space added because of link */
```